net-misc/gns3-server/gns3-server-2.1.20.ebuild

```diff
@@ -0,0 +1,48 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python3_{5,6} )
+
+inherit distutils-r1
+
+DESCRIPTION="GNS3 server to asynchronously manage emulators"
+HOMEPAGE="http://www.gns3.net/"
+SRC_URI="https://github.com/GNS3/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND="
+	>=dev-python/jsonschema-2.6.0[${PYTHON_USEDEP}]
+	<dev-python/aiohttp-2.4.0[${PYTHON_USEDEP}]
+	>=dev-python/aiohttp-2.3.3[${PYTHON_USEDEP}]
+	>=dev-python/aiohttp-cors-0.6.0[${PYTHON_USEDEP}]
+	>=dev-python/jinja-2.7.3[${PYTHON_USEDEP}]
+	>=dev-python/raven-5.23.0[${PYTHON_USEDEP}]
+	>=dev-python/psutil-3.0.0[${PYTHON_USEDEP}]
+	>=dev-python/python-zipstream-1.1.4[${PYTHON_USEDEP}]
+	dev-python/prompt_toolkit[${PYTHON_USEDEP}]
+	<dev-python/async_timeout-3.0.0[${PYTHON_USEDEP}]
+	>=net-misc/ubridge-0.9.14
+"
+DEPEND="dev-python/setuptools[${PYTHON_USEDEP}]"
+
+src_prepare() {
+	default
+
+	# Package installs 'tests' package which is forbidden
```

Comment:
It would be nice to actually remove it *after* testing.

Reply:
(implies enabling tests)

Reply:
this fixes the issues with the install, don't mind if the tests run, just don't want test files being installed.